org.kde.klevernotes.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "org.kde.klevernotes",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.6",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
```

Comment:
you have to cleanup the baseapp, see https://github.com/flathub/io.qt.qtwebengine.BaseApp?tab=readme-ov-file#cleanup

Reply:
Will add this.